Gemfile.lock

```diff
@@ -762,7 +762,7 @@ DEPENDENCIES
   wkhtmltopdf-binary
 
 RUBY VERSION
-   ruby 2.1.5p273
+  ruby 2.1.5p273
 
 BUNDLED WITH
-   1.15.1
+  1.15.1
```

Comment:
@letakeane hi! Just one question, is this the resulting `Gemfile.lock` from `bundle install`? Did you modify it yourself? I'm just curious since Bundler generally adds 3 spaces in both `RUBY VERSION` and `BUNDLED WITH` indentation AFAIK.